web/src/components/PoiDetails.tsx

```diff
@@ -6,7 +6,13 @@ import { useTranslation } from 'react-i18next'
 import { getExternalMapsLink } from 'shared'
 import { PoiModel } from 'shared/api'
 
-import { ExternalLinkIcon, LocationIcon, PoiThumbnailPlaceholderLarge } from '../assets'
+import {
+  AccessibleIcon,
+  ExternalLinkIcon,
+  LocationIcon,
+  NotAccessibleIcon,
```

Comment:
You can you the corresponding icon from mui for web.
For example: https://mui.com/material-ui/material-icons/?selected=Accessible

Reply:
I used Mui for the not accessible one, but since we already have an accessible one, no need to change it, especially since we'll be overwriting it with the Mui ones soon :)

Reply:
In the PR for the new design system, not in main :)